src/main/java/com/dropbox/core/android/AuthActivity.java

```diff
@@ -611,6 +596,7 @@ protected void onNewIntent(Intent intent) {
             secret = intent.getStringExtra(EXTRA_ACCESS_SECRET);
             uid = intent.getStringExtra(EXTRA_UID);
             state = intent.getStringExtra(EXTRA_AUTH_STATE);
+            String extraAuthQueryResults = intent.getStringExtra(EXTRA_AUTH_QUERY_RESULTS);
```

Comment:
Our SDK don't handle these extra results now. Just leave here to show the interface.